components/raftstore/src/store/config.rs

```diff
@@ -250,16 +257,18 @@ impl Default for Config {
             local_read_batch_size: 1024,
             apply_batch_system: BatchSystemConfig::default(),
             store_batch_system: BatchSystemConfig::default(),
+            store_io_pool_size: 2,
             future_poll_size: 1,
             hibernate_regions: true,
             dev_assert: false,
             apply_yield_duration: ReadableDuration::millis(500),
+            perf_level: PerfLevel::Disable,
```

Comment:
I think this can be enabled by default given that perf level is always enabled for read already. /cc @Little-Wallace